io/plugins/eu.esdihumboldt.hale.io.jdbc.msaccess/plugin.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="eu.esdihumboldt.hale.io.jdbc.config">
+      <connection
+            configurer="eu.esdihumboldt.hale.io.jdbc.msaccess.MsAccessConnectionConfigurer"
+            id="eu.esdihumboldt.hale.io.jdbc.msaccess"
+            type="java.sql.Connection">
```

Comment:
Also, right now it would make any connection attempt to other databases fail, because for your connection configurer all connections would be tried to cast to the UCanAccess connection type.